roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -5,6 +5,12 @@
     dest: /usr/local/bin/openshift-node
     mode: 0500
 
+- name: Remove existing systemd service
+  file:
+    path: "/etc/systemd/system/{{ openshift_service_type }}-node.service"
+    state: absent
+  when: not openshift_is_bootstrapped
```

Comment:
This boolean does not exist anymore.

Reply:
This fact is being set in `basic_facts`

The problem here is that `atomic install` won't replace the service file - a better solution would be forcing it to replace it

Reply:
If this is only for atomic, let's switch the boolean to not openshift_is_atomic.

Reply:
It is just for atomic, but a simple booolean in insufficient.

The service file needs to removed when a new system container is being installed - `oc_atomic_container` won't remove existing service file and fail if its not removed.

Added `openshift_is_atomic` to when so that it would happen on Atomic only